testserver/binaries.go

```diff
@@ -0,0 +1,107 @@
+package testserver
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	awsBaseURL      = "https://s3.amazonaws.com/cockroach/cockroach"
+	latestSuffix    = "LATEST"
+	localBinaryPath = "/var/tmp"
```

Comment:
I'm not sure we need to, though I don't know what the conventions are for `/var/tmp`. Can't really find much.